### PR TITLE
Feature/scenario commands distribution

### PR DESCRIPTION
**Pull request by sauroter**

### CODE REVIEW

File: src/main/java/org/varroa/remote/commander/Commander.java [hunk 1]
```diff
@@ -30,24 +30,30 @@
...
     private final @NotNull CommanderConfig commanderConfig;
     private final @NotNull NettyConfig nettyConfig;
+    private final @NotNull Scenario scenario;
+
+    // Logic classes
     private final @NotNull CommanderStateHolder commanderStateHolder;
-    private final @NotNull Provider<CommanderChannelInitializer> commanderChannelInitializer;
     private final @NotNull Provider<ChunkManager> chunkManager;
-    private final @NotNull Scenario scenario;
+    private final @NotNull Provider<CommanderChannelInitializer> commanderChannelInitializer;
```
Comment (SgtSilvio) [inline src/main/java/org/varroa/remote/commander/Commander.java]: imho this should be move up 1 line

File: src/test/java/org/varroa/scenario/ScenarioTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * @author Georg Held
+ */
+class ScenarioTest {
+
+    @Test
+    void name() throws InvalidScenarioException, IOException {
```
Comment (SgtSilvio) [inline src/test/java/org/varroa/scenario/ScenarioTest.java]: please rename the method

File: src/main/java/org/varroa/remote/commander/Commander.java (hunk 1 above)
Comment (sauroter) [reply src/main/java/org/varroa/remote/commander/Commander.java]: why tho ch < co ?

File: src/test/java/org/varroa/scenario/ScenarioTest.java (hunk 2 above)
Comment (sauroter) [reply src/test/java/org/varroa/scenario/ScenarioTest.java]: done

**Pull request merged**